src/components/PasswordField/components/Indicator.js

```diff
@@ -3,23 +3,17 @@
 import React from 'react'
 import classNames from 'classnames'
 
-import { type JInputColor } from 'components/base/JInput/JInput'
-
 import indicatorStyle from './indicator.m.scss'
 
 export type IndicatorStatus = 'red' | 'orange' | 'yellow' | 'green' | 'fetching'
```

Comment:
Offtop:
Maybe make sense get names of statuses from [design documents](https://www.figma.com/file/HqDoUgAbxvgOX5LBqA63H2v0/WEB-Components?node-id=1061%3A24
)?